src/components/structures/RightPanel.tsx

```
@@ -79,6 +80,7 @@ export default class RightPanel extends React.Component<Props, IState> {
 
         this.state = {
             searchQuery: "",
+            windowWidth: window.innerWidth,
```

ESLint

[error] 'window.innerWidth' is restricted from being used. Use UIStore to access window dimensions instead.

(no-restricted-properties)